README.md

```diff
@@ -131,7 +131,7 @@ Breakpoints for the frontend should be set in your browser console.
 
 Be a part of this project! You can run the test using the following.
 
-1. Install dependencies from package.json by running `npm install`
+1. Install dependencies from package.json by running `npm ci`
```

Comment:
I thought the [`clean install`](https://docs.npmjs.com/cli/v8/commands/npm-ci) command is designed for usage in automated environments like CI? 🤔

Reply:
IMHO the design choice of `npm install` updating the dependencies automatically was a mistake, and there should have been separate `npm lock` and `npm install` commands. Why lock dependencies in `package-lock.json` if `npm install` will update (and perhaps break) them next time you try to install? For example, say you know that the deps on master work but a new developer runs `npm install` and that updates the deps to something broken.

I know I'm perhaps overly opinionated/curmudgeonly about this. I can undo the change in the README but I feel strongly about using `npm ci` in the docker image and the CI pipeline.

Reply:
I like NPM CI, good with it